src/shared/molecules/MyDataTable/MyDataTable.tsx

```diff
@@ -130,6 +130,24 @@ export const notifyTotalSizeExeeced = () => {
     key: 'data-panel-size-exceeded',
   });
 };
+const getTypesTrancated = (text: string | string[]) => {
+  let types = '';
+  let typesWithUrl = text;
+  if (isArray(text)) {
+    types = text
+      .map(item => (isValidUrl(item) ? item.split('/').pop() : item))
+      .join('\n');
+    typesWithUrl = text.join('\n');
+  } else if (isString(text) && isValidUrl(text)) {
+    types = text.split('/').pop() ?? '';
```

Comment:
The `?? ''` is not needed here, right? If there is no `/` in `text` we'll just get the whole string so the program will never reach `?? ''`.